force-app/main/default/lwc/geFormRenderer/geFormRenderer.js

```diff
@@ -988,23 +990,23 @@ export default class GeFormRenderer extends LightningElement{
     }
 
     populateFormStateWithRDInfo(elevateBatchItem) {
-        const isSuccessful = elevateBatchItem.status === 'Active';
+        const isSuccessful = elevateBatchItem.status === COMMITMENT_INACTIVE_STATUS &&
```

Comment:
I just want to confirm that this will be the response even when the RD is processed right away. I assume it becomes 'Active' after the first payment is paid, which will happen async.

Reply:
yeah the "Active" status will be set by BDI